Aztec/Classes/ElementConverters/Implementations/BRElementConverter.swift

```diff
@@ -0,0 +1,7 @@
+import UIKit
+
+class BRElementConverter: ElementConverter {
```

Comment:
(Nipticky): We're using (nearly all over!) something like this:

```
// MARK: - FormatBar
//
open class FormatBar: UIView
```

And in few other spots, something like this:

```
/// Parses an attributed string into an HTML tree.
///
class AttributedStringParser {
```

I think we haven't really settled on anything there, but maybe (maybe) pick one of them two, and add it to the new classes, just for consistency?